ehrql/tables/beta/tpp.py

```diff
@@ -231,80 +247,180 @@ class clinical_events(EventFrame):
 
 @table
 class ec(EventFrame):
+    """
+    Emergency care admissions data is provided via the NHS Secondary Uses Service.
+
+    This table gives core details of admissions.
+    """
+
     ec_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Unique identifier used across the EC tables.",
     )
     arrival_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date the patient self presented at the accident & emergency department, or arrived in an ambulance at the accident & emergency department.",
     )
     sus_hrg_code = Series(
         str,
-        description="TODO",
+        description="The core Healthcare Resource Group (HRG) code derived by sus+, as a result of HRG grouping, and used for tariff application.",
```

Comment:
[throughout] probably remove mention of the HRG grouper/grouping process as it won't mean much to most people